### ISSUE TITLE
Docker container not connecting:  Whoops! Connection failed. Please try again. If the problem persists, contact our customer support.

### ISSUE THREAD

**Issue by shaftspanner** [bubuntux/nordvpn#118]
**Describe the bug**
Docker container fails to start - Exits with error message "Whoops! Connection failed.  Please try again.  If the problem persists, contact our customer support"

I was initially getting DNS errors so I added the DNS environment label which solved that problem but the contain still stops with this error message.

I initially tried in docker-compose but was getting the same problems so went to docker CLI and used the basic example from README.MD

**To Reproduce using docker CLI**
```
sudo docker run -ti --cap-add=NET_ADMIN --cap-add=SYS_MODULE --device /dev/net/tun --name vpn \
            --sysctl net.ipv4.conf.all.rp_filter=2 \
            -e USER=xxxxx@live.co.uk -e PASS='xxxxx' \
            -e CONNECT=United_States -e TECHNOLOGY=NordLynx -e DNS=8.8.8.8,8.8.4.4 \
			-e DEBUG=on -d bubuntux/nordvpn

```

**Expected behavior**
The container starts, connects and keeps running

**Logs**
```
     
-P INPUT DROP

-P FORWARD DROP

-P OUTPUT DROP

-A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT

-A INPUT -i lo -j ACCEPT

-A INPUT -s 172.17.0.0/16 -j ACCEPT

-A FORWARD -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT

-A FORWARD -i lo -j ACCEPT

-A FORWARD -d 172.17.0.0/16 -j ACCEPT

-A FORWARD -s 172.17.0.0/16 -j ACCEPT

-A OUTPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT

-A OUTPUT -o lo -j ACCEPT

-A OUTPUT -o tap+ -j ACCEPT

-A OUTPUT -o tun+ -j ACCEPT

-A OUTPUT -m owner --gid-owner 104 -j ACCEPT

-A OUTPUT -d 172.17.0.0/16 -j ACCEPT



nat:

-P PREROUTING ACCEPT

-P INPUT ACCEPT

-P OUTPUT ACCEPT

-P POSTROUTING ACCEPT

-A POSTROUTING -o tap+ -j MASQUERADE

-A POSTROUTING -o tun+ -j MASQUERADE



mangle:

-P PREROUTING ACCEPT

-P INPUT ACCEPT

-P FORWARD ACCEPT

-P OUTPUT ACCEPT

-P POSTROUTING ACCEPT



raw:

-P PREROUTING ACCEPT

-P OUTPUT ACCEPT



security:

-P INPUT ACCEPT

-P FORWARD ACCEPT

-P OUTPUT ACCEPT






\2020/11/20 10:07:21 [Error] disabling IPv6: seting net.ipv6.conf.all.disable_ipv6 to 1: exit status 255: sysctl: setting key "net.ipv6.conf.all.disable_ipv6": Read-only file system




  

-2020/11/20 10:07:21 POST_CONNECT system info:

Routing Table:

default via 172.17.0.1 dev eth0 

172.17.0.0/16 dev eth0 proto kernel scope link src 172.17.0.4 



IP Rules:

0:	from all lookup local 

32766:	from all lookup main 

32767:	from all lookup default 



IP Tables:

filter:

-P INPUT DROP

-P FORWARD DROP

-P OUTPUT DROP

-A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT

-A INPUT -i lo -j ACCEPT

-A INPUT -s 172.17.0.0/16 -j ACCEPT

-A FORWARD -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT

-A FORWARD -i lo -j ACCEPT

-A FORWARD -d 172.17.0.0/16 -j ACCEPT

-A FORWARD -s 172.17.0.0/16 -j ACCEPT

-A OUTPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT

-A OUTPUT -o lo -j ACCEPT

-A OUTPUT -o tap+ -j ACCEPT

-A OUTPUT -o tun+ -j ACCEPT

-A OUTPUT -m owner --gid-owner 104 -j ACCEPT

-A OUTPUT -d 172.17.0.0/16 -j ACCEPT



nat:

-P PREROUTING ACCEPT

-P INPUT ACCEPT

-P OUTPUT ACCEPT

-P POSTROUTING ACCEPT

-A POSTROUTING -o tap+ -j MASQUERADE

-A POSTROUTING -o tun+ -j MASQUERADE



mangle:

-P PREROUTING ACCEPT

-P INPUT ACCEPT

-P FORWARD ACCEPT

-P OUTPUT ACCEPT

-P POSTROUTING ACCEPT



raw:

-P PREROUTING ACCEPT

-P OUTPUT ACCEPT



security:

-P INPUT ACCEPT

-P FORWARD ACCEPT

-P OUTPUT ACCEPT






  
Whoops! We couldn't connect you to 'United_States'. Please try again. If the problem persists, contact our customer support.

+ exit 1
```

**Additional context**
Ubuntu 18.04.5 LTS
Docker 19.03.13
Architecture AMD64

**Comment by ale-saglia** [comment]
Seems to be some problem with the lates docker image (see  #103)
A quick fix is to roll back to 3.7.4 changing bubuntux/nordvpn to bubuntux/nordvpn:3.7.4

**Comment by slimcdk** [comment]
When will this be resolved? I seems to persist in latest version
